Keyboard.js

```diff
@@ -8,6 +8,10 @@ define([
 	'./util/misc',
 	'dojo/_base/sniff'
 ], function (declare, aspect, domClass, on, lang, has, miscUtil) {
+	
+	var hasPointer = has("pointer"),
+            hasMSPointer = hasPointer && hasPointer.slice(0, 2) === "MS",
```

Comment:
Dojo 1.10 and newer have an MS pointer event feature test, https://github.com/dojo/dojo/commit/b25b080d6fb0e1b9e64ba48e303d207614f80ada , so we could use that. In order to support Dojo versions prior to 1.10, we could replicate that test in a dgrid/util/has-pointer module?